src/Arcus.WebApi.Security/Model/ClaimCheck.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections.Generic;
+
+namespace Arcus.WebApi.Security.Model
+{
+    public class ClaimCheck : Dictionary<string, string>
+    {
+    }
```

Comment:
In my opinion, `ClaimCheck` should be a simple value type.

Something that looks like 

```csharp
public class ClaimCheck
{
       public string Claim {get;}
       public string Value {get;}
}
```

Maybe even implement IEquatable on it.

Reply:
I also thought about this solution. However, a Dictionary gives us the implicit validation of having unique values. That is my only reason, otherwise, if it is a value type or not, the comparison logic will be the same, only a matter of definition of the object type.

Reply:
You do have a point regarding the uniqueness, but I think that creating a type which inherits from Dictionary is just to implicit.   If you want to use a dictionary for this, then I'd suggest to pass in a Dictionary and not a type that inherits from Dictionary since that is not clear to the consumer of the library.

Reply:
done